testing-modules/assertion-libraries/src/main/java/com/baeldung/assertj/exceptions/CityUtils.java

```diff
@@ -0,0 +1,17 @@
+package com.baeldung.assertj.exceptions;
+
+import java.util.Arrays;
+import java.util.List;
+
+public final class CityUtils {
+
+    private static final List<String> CITIES = Arrays.asList("Tamassint", "Londn", "Madrid", "New york");
+
+    public static String search(String searchedCity) {
+        return CITIES.stream()
+          .filter(city -> searchedCity.equals(city))
```

Comment:
We can simplify with a method reference:
```suggestion
          .filter(searchedCity::equals)
```

Reply:
Done